src/components/AttachResult/AttachResult.ts

```diff
@@ -50,6 +54,10 @@ export class AttachResult extends Component {
         detachCaption: ComponentOptions.buildStringOption({
             defaultValue: l(`${AttachResult.ID}_Detach`)
         }),
+        articleIdField: ComponentOptions.buildStringOption({
+            defaultValue: 'permanentid'
```

Comment:
Why should it take this default value?

Reply:
that's the default ML uses to compute things so it's the most relevant field normally (ml-wise)